sdk/python/kfp/components/_python_op.py

```diff
@@ -45,6 +45,47 @@ def _python_function_name_to_component_name(name):
     return re.sub(' +', ' ', name.replace('_', ' ')).strip(' ').capitalize()
 
 
+def _capture_function_code_using_source_copy(func) -> str:
```

Comment:
This is not used? If so, can we delete the function?

Reply:
Are you sure? I wanted to keep a way (undocumented) to use the old method for a little bit of time instead of completely removing the old implementation.
The new implementation should support everything that the old implementation supported, but the source code is no longer visible.

But if you think it's better to delete the old code, then I can do that.

Reply:
We should remove it to avoid confusion for developers. Besides, we can always revert to the old versions because github has the records. WDYT?

Reply:
OK. I can do that.
AT first I wanted to expose the  "code extractor" parameter in the API, but later decided not to expose it publicly.
I guess we can remove the code.

Reply:
Thanks

Reply:
Done.